libr/core/cmd_debug.c

```diff
@@ -1102,13 +1101,7 @@ static int cmd_debug_map(RCore *core, const char *input) {
 		break;
 	case 'h': // "dmh"
 #if __linux__ && __GNU_LIBRARY__ && __GLIBC__ && __GLIBC_MINOR__
-		if (SZ == 4) {
```

Comment:
this is wrong, we should call _32 or _64, like it was before. and not call the GH() one